orchestra/static/orchestra/timing/time-entry.js

```diff
@@ -96,6 +96,7 @@
           }
         }
         entry.date = moment(entry.date).startOf('day');
+        entry.assignment = data.assignment || undefined;
```

Comment:
I'm not sure if it comes back as `null` or `undefined`, and could imagine that changing our implementation server side could change this—since I check for it elsewhere, I didn't want to check for both possibilities.